network/network_linux.go

```diff
@@ -215,11 +216,13 @@ func getMajorVersion(version string) (int, error) {
 	return 0, fmt.Errorf("[net] Error getting major version")
 }
 
-func isGreaterOrEqaulUbuntuVersion(versionToMatch int) bool {
+// checkUbuntuVersion() checks ubuntu version and whether it's equal or greater than versionToMatch
+// return ubuntu version if it's equal or greater than versionToMatch; otherwise return 0
+func checkUbuntuVersion(versionToMatch int) (int, bool) {
```

Comment:
can we split this to api to 2? one to get ubuntu version and other to compare version